lib/collection/src/shard/remote_shard.rs

```diff
@@ -1,12 +1,34 @@
+use crate::operations::payload_ops::PayloadOps;
+use crate::operations::point_ops::{PointInsertOperations, PointOperations};
+use crate::operations::FieldIndexOperations;
 use crate::shard::{PeerId, ShardId, ShardOperation};
 use crate::{
-    CollectionId, CollectionInfo, CollectionResult, CollectionSearcher, CollectionUpdateOperations,
-    PointRequest, Record, SearchRequest, UpdateResult,
+    CollectionError, CollectionId, CollectionInfo, CollectionResult, CollectionSearcher,
+    CollectionUpdateOperations, PointRequest, Record, SearchRequest, UpdateResult,
+};
+use api::grpc::conversions::payload_to_proto;
+use api::grpc::qdrant::collections_internal_client::CollectionsInternalClient;
+use api::grpc::qdrant::points_internal_client::PointsInternalClient;
+use api::grpc::qdrant::points_selector::PointsSelectorOneOf;
+use api::grpc::qdrant::{
+    ClearPayloadPoints, ClearPayloadPointsInternal, CreateFieldIndexCollection,
+    CreateFieldIndexCollectionInternal, DeleteFieldIndexCollection,
+    DeleteFieldIndexCollectionInternal, DeletePayloadPoints, DeletePayloadPointsInternal,
+    DeletePoints, DeletePointsInternal, GetCollectionInfoRequest, GetCollectionInfoRequestInternal,
+    GetPoints, GetPointsInternal, PointsIdsList, PointsSelector, ScrollPoints,
+    ScrollPointsInternal, SearchPoints, SearchPointsInternal, SetPayloadPoints,
+    SetPayloadPointsInternal, UpsertPoints, UpsertPointsInternal,
 };
```

Comment:
```suggestion
use api::grpc::conversions::payload_to_proto;
use api::grpc::qdrant::{points_selector::PointsSelectorOneOf, points_internal_client::PointsInternalClient, collections_internal_client::CollectionsInternalClient,
    ClearPayloadPoints, ClearPayloadPointsInternal, CreateFieldIndexCollection,
    CreateFieldIndexCollectionInternal, DeleteFieldIndexCollection,
    DeleteFieldIndexCollectionInternal, DeletePayloadPoints, DeletePayloadPointsInternal,
    DeletePoints, DeletePointsInternal, GetCollectionInfoRequest, GetCollectionInfoRequestInternal,
    GetPoints, GetPointsInternal, PointsIdsList, PointsSelector, ScrollPoints,
    ScrollPointsInternal, SearchPoints, SearchPointsInternal, SetPayloadPoints,
    SetPayloadPointsInternal, UpsertPoints, UpsertPointsInternal,
};
```

Reply:
Nice catch :+1: 

Done in https://github.com/qdrant/qdrant/pull/505/commits/5151de2032701f64a90bfc8acbb0a94a0d369edd